src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF2009.ConsiderConventionalIdentifierValues_Invalid.sarif

```diff
@@ -17,6 +17,11 @@
           ]
         }
       },
+      "versionControlProvenance": [
+        {
+          "repositoryUri": "https://github.com/microsoft/sarif-sdk"
+        }
+      ],
```

Comment:
>], [](start = 6, length = 2)

This is right, but why don't _all_ the tests need it?

Reply:
Answer: because this is the only other "note"-level rule.

---
In reply to: [447908317](https://github.com/microsoft/sarif-sdk/pull/1947#discussion_r447908317) [](ancestors = 447908317)